crates/hyperswitch_connectors/src/connectors/digitalvirgo.rs

```diff
@@ -168,9 +234,9 @@ impl ConnectorIntegration<Authorize, PaymentsAuthorizeData, PaymentsResponseData
     fn get_url(
         &self,
         _req: &PaymentsAuthorizeRouterData,
-        _connectors: &Connectors,
+        connectors: &Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
-        Err(errors::ConnectorError::NotImplemented("get_url method".to_string()).into())
+        Ok(format!("{}{}", self.base_url(connectors), "/payment"))
```

Comment:
Please make the same changes in all get_url functions.

```suggestion
        Ok(format!("{}/payment", self.base_url(connectors)))
```